packages/dashboard/docker/rmf/docker-compose.yml

```
@@ -2,4 +2,4 @@ version: "3"
 services:
   build:
     build: .
-    image: docker.pkg.github.com/osrf/romi-dashboard/rmf:nightly
+    image: docker.pkg.github.com/osrf/rmf-web/e2e/rmf:nightly
```

This doesn't look right. Shouldn't it be
```suggestion
FROM docker.pkg.github.com/osrf/rmf-web/rmf:nightly
```